src/index.html

```diff
@@ -1,15 +1,215 @@
+
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <section class="box">
+          <label for="surname" class="form-field">
+            Surname:
+            <input type="text"
+              id="surname"
+              name="surname"
+              autocomplete="off"
+              required
+            >
+          </label>
+        </section>
+
+        <section class="box">
+          <label for="name" class="form-field">
+            Name:
+            <input type="text"
+              id="name"
+              name="name"
+              required
+              autocomplete="off"
+            >
+          </label>
+        </section>
+
+        <section class="box">
+          <label for="age" class="form-field">
+            How old are You?
+            <input type="number"
+              id="age"
+              name="age"
+              value="12"
+              min="5"
+              max="100"
+            >
+          </label>
+        </section>
+
+        <section class="box">
+          <label for="date" class="form-field">
+            Full date of birth:
+            <input type="date"
+              id="date"
+              name="date"
+            >
+          </label>
+        </section>
+
+        <section class="box">
+          <label for="terms" class="form-field">
+            I accept the term of the agreement
+            <input type="checkbox"
+              id="terms"
+              name="terms"
+            >
+          </label>
+        </section>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <section class="box">
+          <label for="email" class="form-field">
+            E-mail:
+            <input type="email"
+              id="email"
+              name="email"
+              placeholder="email@example.com"
+              required
+              autocomplete="off"
+            >
+          </label>
+        </section>
+
+        <section class="box">
+          <label for="password"
+            class="form-field">
+            Password:
+            <input type="password"
+              id="password"
+              name="password"
+              min="8"
+              max="20"
+              required
+              autocomplete="off"
+              minlength="6"
+              maxlength="20"
+            >
+          </label>
+        </section>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <section class="section box">
+          Do you love cats?
+          <label for="Yes">
+            <input type="radio"
+              id="Yes"
+              name="cats-yes"
+              value="Yes"
+            >
+            Yes
+          </label>
+          <label for="No">
+            <input type="radio"
+              id="No"
+              name="cats-no"
+              value="No"
+            >
+            No
+          </label>
```

Comment:
give more specific names for id, not just yes or no
also, radio buttons should have the same name, otherwise I can check both of them

![image](https://user-images.githubusercontent.com/57598825/112445761-29089980-8d58-11eb-848a-c5778130d768.png)


```suggestion
          <label for="cats-yes">
            <input type="radio"
              id="cats-yes"
              name="love-cats"
              value="Yes"
            >
            Yes
          </label>
          <label for="cats-no">
            <input type="radio"
              id="cats-no"
              name="love-cats"
              value="no"
            >
            No
          </label>
```